source/common/upstream/outlier_detection_impl.cc

```diff
@@ -33,7 +34,12 @@ void DetectorHostSinkImpl::uneject(SystemTime unejection_time) {
   last_unejection_time_.value(unejection_time);
 }
 
+void DetectorHostSinkImpl::updateCurrentSRBucket() {
+  sr_accumulator_bucket_.store(sr_accumulator_.getCurrentWriter());
```

Comment:
Everywhere in this entire change where you do just SR switch to some variant of "Success rate"